README.md

```diff
@@ -58,9 +58,32 @@ In order to test the provider, you can simply run `make test`.
 $ make test
 ```
 
+Note that the above command runs acceptance tests by replaying pre-recorded API responses
+(cassettes) stored in `datadog/cassettes/`. When tests are modified, the cassettes need
+to be re-recorded.
+
+In order to make tests cassette friendly, it's necessary to ensure that resources always get
+manipulated in a predictable order. When creating a testing Terraform config that defines multiple
+resources at the same time, you need to set inter-resource dependencies (using `depends_on`)
+in such a way that there is only one way for Terraform to manipulate them. For example, given
+resources A, B and C in the same config string, you can achieve this by making A depend on B
+and B depend on C. See [PR #442](https://github.com/terraform-providers/terraform-provider-datadog/pull/442)
+for an example of this.
+
+*Note:* Recording cassettes creates/updates/destroys real resources. Never run this on
+a production Datadog organization.
+
```

Comment:
It looks like a duplicate of lines 85-86
```suggestion
```

Reply:
It's not a duplicate. This line is about cassettes, lines 85-86 are about integration tests. These are 2 distinct commands that both would use keys and do various changes on the DD org, so let's include explicit warning for both of them.